javascript/chronometer.js

```diff
@@ -1,26 +1,47 @@
 class Chronometer {
   constructor() {
+    this.currentTime = 0;
+    this.intervalId = 0;
     // ... your code goes here
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (callback) {
+      callback();
```

Comment:
very clever to use the condition that will evaluate if it is not undefined first.